maintainers/maintainer-list.nix

```diff
@@ -2863,6 +2863,12 @@
     githubId = 30512529;
     name = "Evils";
   };
+  ewok = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add ewok``.

Reply:
This is done